backend/main/src/Services/CreateNewUserService/CreateANewUserService.ts

```diff
@@ -0,0 +1,95 @@
+import { CreateUserController } from "src/User/Controllers/CreateUserController";
+import { CreateContainerController } from "src/Group/Controllers/CreateContainerController";
+import { CreateGroupController } from "src/Group/Controllers/CreateGroupController";
+import {
+	BadRequestStatus,
+	Controller,
+	ForbiddenStatus,
+	InternalServerErrorStatus,
+} from "src/Shared";
+import { NishikiDynamoDBClient } from "src/Shared/Adapters/DB/NishikiTableClient";
+import { IGroupRepository } from "src/Group/Domain/IGroupRepository";
+import { GroupId } from "src/Group/Domain/Entities/Group";
+
+interface ICreateANewUser {
+	emailAddress: string;
+	name?: string;
+}
+
+/**
+ * Create a new user, container, and group.
+ */
+export class CreateANewUserService extends Controller<ICreateANewUser> {
+	constructor(
+		readonly createUser: CreateUserController,
+		readonly createGroup: CreateGroupController,
+		readonly createContainer: CreateContainerController,
+		readonly groupRepository: IGroupRepository,
+		readonly nishikiDynamoDBClient: NishikiDynamoDBClient,
+	) {
+		super();
+	}
+
+	async handler(input: ICreateANewUser) {
+		// create a new user
+		const createUserResult = await this.createUser.execute(input);
+		if (!(createUserResult.status === "CREATED" && createUserResult.body)) {
+			return createUserResult as BadRequestStatus | InternalServerErrorStatus;
+		}
```

Comment:
IMO, I think the Service should call Usecases instead of Controllers because the Service that extends the Controller class shouldn't call the same layer's class. And if it calls Usecases, it makes it easier to handle errors.
But it might have Cons, so let me know if you find any problems with the implementation with Usecases. 
```suggestion
		if (!createUserResult.ok) {
			return this.badRequest(createUserResult.error.message);
		}
```